test.js

```diff
@@ -163,3 +165,114 @@ test(`use relative path with '..' chars`, async t => {
 	const {stdout} = await m(pathViaParentDir, ['foo']);
 	t.is(stdout, 'foo');
 });
+
+test('err.killed is true if process was killed directly', async t => {
+	const cp = m('forever');
+
+	setTimeout(function () {
+		cp.kill();
+	}, 100);
+
+	const err = await t.throws(cp);
+
+	t.true(err.killed);
+});
+
+// TODO: Should this really be the case, or should we improve on child_process?
+test('err.killed is false if process was killed indirectly', async t => {
```

Comment:
I'm with you on this James. If it's killled, it's killed, regardless of how it was done. And if it is killed, jou woild expect that flag to be true.